lib/ace/ext/papeeria/highlighter_test.js

```diff
@@ -0,0 +1,137 @@
+define(function(require, exports, module) {
+
+    var EditSession = require("ace/edit_session").EditSession;
+    var Editor = require("ace/editor").Editor;
+    var MockRenderer = require("ace/test/mockrenderer").MockRenderer;
+    var assert = require("ace/test/assertions");
+    var highlighter = require("ace/ext/papeeria/highlighter");
+    var Range = require("ace/range").Range
+
+    getRangeHighlightText = function(session) {
+        var allMarkers = session.getMarkers();
+        for (var key in allMarkers) 
+           if (allMarkers[key].clazz == "ace_error-marker")
+              return allMarkers[key].range
+        return null
+    }
+
+    getPositionHighlightText = function(session) {
```

Comment:
did you mean that `getRange` is `getMismatchRange` and `getPosition` is getMatchRange` ?